sdk/tools/kbdtool/main.c

```diff
@@ -61,7 +61,8 @@ INT
 main(INT argc,
      PCHAR* argv)
 {
-    ULONG i, ErrorCode, FailureCode;
+    int i;
```

Comment:
`INT`

Reply:
`INT` and `int` have been used synonymously throughout our entire code base, and also by other Windows developers, so I see no reason for such a change.
We really don't have to reinvent `int i` just because MS defines an uppercase type for it.